packages/SwingSet/test/test-vpid-liveslots.js

```diff
@@ -239,14 +239,23 @@ async function doVatResolveCase1(t, mode) {
   // then it should send 'two'. For now it should cite the same promise ID,
   // but in the future that vpid will have been retired, and we should see a
   // different one
+  let expectedArg = expectedP3;
```

Comment:
let's put a `const RETIRE_VPIDS = ` with the mode check here, and switch on it (instead of the mode) later, to be readably parallel with the Case2/3 test below

Also the comment needs updating